packages/samples/vuejs/.eslintrc.cjs

```diff
@@ -0,0 +1,15 @@
+/* eslint-env node */
+require('@rushstack/eslint-patch/modern-module-resolution');
+
+module.exports = {
+  root: true,
+  extends: [
+    'plugin:vue/vue3-essential',
+    'eslint:recommended',
+    '@vue/eslint-config-typescript/recommended',
+    '@vue/eslint-config-prettier',
```

Comment:
Since this uses prettier and people may clone this, I think it would be neat to add prettier to the recommended extensions and to add format on save by default.

Reply:
I'm adding that in the next PR!